internal/runner/client/client_local.go

```diff
@@ -220,6 +284,30 @@ func shellID() (int, bool) {
 	return i, true
 }
 
+// TODO(cepeda): Session is not working
```

Comment:
fyi, one of the big difference between local and remote runner impls is that session are ad-hoc versus "persistent". the only exception is the `tui`. however, that's a long-running process.